### PR TITLE
Prepare 3.0 release

### PR DESCRIPTION
**Pull request by JialinXin**
1. Upgrade build version to 3.0.100
2. Deprecate PackageIconUrl: https://aka.ms/deprecateIconUrl 
3. Upgrade blazor app sample to latest template 
4. Fix tests

2-4 relate to 1.

### CODE REVIEW

File: src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj [hunk 1]
```diff
@@ -46,4 +43,11 @@
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="$(MicrosoftAspNetCoreSignalRPackageVersion)" />
   </ItemGroup>
+
+  <!-- Specifying signing for multiple files in a package. -->
+  <ItemGroup>
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj]: move to `directory.Build.props`?

File: test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs [hunk 2]
```diff
@@ -357,6 +358,8 @@ public override ServiceEndpoint GetNegotiateEndpoint(HttpContext context, IEnume
                 {
                     context.Response.StatusCode = 400;
                     var response = Encoding.UTF8.GetBytes("Invalid request");
+                    // In latest DefaultHttpContext, response body is set to null
+                    context.Response.Body = new MemoryStream();
```
Comment (vicancy) [inline test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs]: check if it is null first?

Comment (JialinXin) [reply test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs]: Directly access Body to check null will return ObjectNullReference exception.

Comment (JialinXin) [reply test/Microsoft.Azure.SignalR.Tests/NegotiateHandlerFacts.cs]: Looks in AspNetCore tests they just new MemoryStream() for the DefaultHttpContext(). Let's follow it.
https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/common/Http.Connections/test/ServerSentEventsTests.cs#L119

File: src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj (hunk 1 above)
Comment (JialinXin) [reply src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj]: I thought different packages have different dlls. Looks move the common Directory.Build.props also works. Nice catch.

**Pull request merged**